tests/test_utils/test_sft.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+from pyfstat import Writer
+from pyfstat.utils import get_sft_as_arrays
+
+
+def test_get_sft_as_arrays(tmp_path):
+
+    writer_kwargs = {
+        "sqrtSX": 1,
+        "Tsft": 1800,
+        "timestamps": {
+            "H1": np.array([1, 2, 3, 6, 7, 10, 12]),
+            "L1": np.array([14, 25, 36, 47]),
+        },
```

Comment:
Those are the most overlapping SFTs I've ever seen. 1980 sure was wild times. :wink: A bit surprised that this doesn't trigger any funky errors, but ok. Still, in the sense of "tests are also a bit of an example documentation of how to use the code", maybe put it something more "normal"?